api/v1alpha1/iochaos_types.go

```diff
@@ -125,11 +81,28 @@ type IoChaosSpec struct {
 	// Methods defines the I/O methods for injecting I/O chaos action.
 	// default: all I/O methods.
 	// +optional
-	Methods []string `json:"methods,omitempty"`
+	Methods []IoMethod `json:"methods,omitempty"`
+
+	// Percent defines the percentage of injection errors and provides a number from 0-100.
+	// default: 100.
+	// +optional
+	Percent int `json:"percent,omitempty"`
+
+	// VolumePath represents the mount path of injected volume
+	VolumePath string `json:"volumePath"`
```

Comment:
`VolumePath` is a newly added field. It's used to specify the path of target volume. @g1eny0ung